smarts/sstudio/genscenario.py

```diff
@@ -242,12 +246,14 @@ def gen_scenario(
         db_conn, scenario.traffic, artifact_paths, obj_hash, map_needs_rebuild
     ):
         with timeit("traffic", logger.info):
-            for name, traffic in scenario.traffic.items():
+
+            for iteration, (name, traffic) in enumerate(scenario.traffic.items()):
+                derived_seed = seed + iteration
                 gen_traffic(
                     scenario=scenario_dir,
                     traffic=traffic,
                     name=name,
-                    seed=seed,
+                    seed=derived_seed,
```

Comment:
Perhaps we should wrap the seed here (at the point where it is introduced) instead of later where it is used in `generator.plan_and_save()`.

Reply:
I disagree, if we wrap it at this level then the implementation will affect how we use this method.